CONTRIBUTING.md

```diff
@@ -26,3 +26,34 @@ information on using pull requests.
 
 This project follows [Google's Open Source Community
 Guidelines](https://opensource.google/conduct/).
+
+## Commit Message Conventions
+- Start with a short summary (50 characters or less) of the changes made.
+- Use the present tense and imperative mood.
+- Separate the summary from the body of the message with a blank line.
+- Use the body to explain what and why changes were made, as well as any necessary details.
+- Addionally, you can consider using [semantic commit messages](https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716?permalink_comment_id=3867882) like "feat:","docs:",etc. which will provide additional context to the commit message.
```

Comment:
space after comma.

Reply:
@rudrakshkarpe this is not addressed yet.

Reply:
> @rudrakshkarpe this is not addressed yet.

I apologize for the delay, I just fixed it. Please take a look at it.